spec/graphql/queries/set_query_spec.rb

```diff
@@ -0,0 +1,65 @@
+describe 'getCollection' do
+  before(:each) do
+    FactoryGirl.build(:meta_datum_title_with_collection).save(validate: false)
+    # When I try to just create :meta_datum_title_with_collection I get
```

Comment:
@kooropatfa to fix this issue add `config.active_record.belongs_to_required_by_default = false` to `config/application.rb` after `load_defaults` call